README.md

```diff
@@ -49,13 +49,13 @@ Using an unsupported format will throw an exception.
 <?php
 
 // Make sure to either deserialize the Event/Place from JSON, or set the necessary properties through setCalendarType() etc.
-$event = new Event();
+$offer = new \CultuurNet\CalendarSummaryV3\Offer\Offer::fromJsonLd('JSONLD_STRING');
     
 // This will format the calendar info of $event in an medium HTML output 
```

Comment:
```// This will format the calendar info of $offer in an medium HTML output```

(Can't make a suggestion that you can commit directly for some reason)